### PR TITLE
rest: add swagger doc for tenants

### PR DESCRIPTION
**Pull request by jiazhai**
rest: add swagger doc for tenants

### CODE REVIEW

File: pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TenantInfo.java [hunk 1]
```diff
@@ -20,20 +20,31 @@
...
 import lombok.Data;
 
 @Data
+@ApiModel(value = "TenantInfo", description = "Information of adminRoles and allowedClusters for tenant")
 public class TenantInfo {
     /**
      * List of role enabled as admin for this tenant
      */
+    @ApiModelProperty(
+        value = "Comma separated list of auth principal allowed to administrate the tenant. If empty the current set of roles won't be modified",
```
Comment (merlimat) [inline pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TenantInfo.java]: I think the check might only be in the CLI tool. Not 100% sure

File: pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TenantInfo.java [hunk 2]
```diff
@@ -20,20 +20,31 @@
...
+        name = "adminRoles"
+    )
     private Set<String> adminRoles;
 
     /**
      * List of clusters this tenant is restricted on
      */
+    @ApiModelProperty(
+        value = "Comma separated allowed clusters. If omitted, the current set of clusters will be preserved",
```
Comment (merlimat) [inline pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TenantInfo.java]: Same here, not sure the doc is correct for REST point of view

**Comment by merlimat** [comment]
@jiazhai 

```
[pulsar_precommit_java8] $ /bin/bash -xe /tmp/jenkins4167227195723580361.sh
+ src/check-binary-license ./distribution/server/target/apache-pulsar-2.4.0-SNAPSHOT-bin.tar.gz
swagger-annotations-1.5.21.jar unaccounted for in lib/presto/LICENSE
```

**Comment by jiazhai** [comment]
Thanks @merlimat for the comments.
updated the license file.

regarding `ApiModelProperty`, it is able to work for rest.
image

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
@merlimat would you please help take a look again?

**Comment by merlimat** [comment]
> regarding ApiModelProperty, it is able to work for rest.

I mean the description itself. I'm not sure of the `If empty the current set of roles won't be modified` part. If that it's on the REST handler or just implemented in the CLI tool.

**Comment by jiazhai** [comment]
Thanks @merlimat. You are right, in rest, we should provide both adminRoles and allowedClusters.

**Comment by jiazhai** [comment]
run java8 tests

**Pull request merged**